src/query/interpreter.hpp

```diff
@@ -51,7 +51,7 @@ extern const Event FailedQuery;
 namespace memgraph::query {
 
 inline constexpr size_t kExecutionMemoryBlockSize = 1UL * 1024UL * 1024UL;
-inline constexpr size_t kExecutionPoolMaxBlockSize = 32768UL;  // 2 ^ 15
+inline constexpr size_t kExecutionPoolMaxBlockSize = 2048UL;  // 2 ^ 11
```

Comment:
Could you please write down the reasons behind this change also?

Reply:
Yes. I made some comparisons, and there were no memory spikes when I reduced `max_block_size` to be smaller than one `row` that wants to allocate memory. To be specific, I tried with rows that had 100 properties with 10 chars for each property, which resulted in 100*10*~8 = 8000 bytes which needed to be allocated for each `row`, and there were no memory spikes when unpooled memory was used in such cases. 

And it seems to me that having a max of 32kB * 8 allocated memory when we have really big rows, would result in some kind of issues, so I reduced `max_block_size` as it seems unnecessary for the threshold to be so big when  `unpooled_memory` can be used without suffering memory issues.